package.json

```diff
@@ -26,6 +26,7 @@
     "generate-android-snapshot": "./bin/generate-android-snapshot"
   },
   "dependencies": {
+    "semver": "^5.4.1",
```

Comment:
An exact version would be better IMO - `^5.4.1` could cause problems in case `semver` releases 5.5.0 which has breaking changes (which shouldn't happen but can)